spec/services/nytimes_service_spec.rb

```diff
@@ -9,22 +9,24 @@
 			end
 
 			expect(Article.first.title).to(
-				eq("California Imposes Water Restrictions, in First for State")
+				eq("A Nuclear Deal Built on Coffee, Compromise and No Sleep")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.